tests/e2e/cucumber/steps/ui/shares.ts

```diff
@@ -145,3 +145,23 @@ When(
     await pageObject.navigate()
   }
 )
+
+Then(
+  /"([^"]*)" should( not | )be able to see the following shares$/,
+  async function (
+    this: World,
+    stepUser: string,
+    condition: string,
+    stepTable: DataTable
+  ): Promise<void> {
+    const shouldExist = condition.trim() !== 'not'
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const shareObject = new objects.applicationFiles.Share({ page })
+    for (const { resource, owner } of stepTable.hashes()) {
+      const isAcceptedSharePresent = await shareObject.isAcceptedSharePresent(resource, owner)
+      expect(isAcceptedSharePresent, '${resource} does not exist in accepted share').toBe(
```

Comment:
So it works?

Reply:
it was never a question it work or not ?